test/data-source.test.js

```diff
@@ -103,7 +102,9 @@ describe('DataSource', function() {
         var fn = scope[name];
         var actuallyEnabled = Model.getRemoteMethod(name);
         assert(fn, name + ' should be defined!');
-        assert(actuallyEnabled === isRemoteEnabled, name + ' ' + (isRemoteEnabled ? 'should' : 'should not') + ' be remote enabled');
+        assert(actuallyEnabled === isRemoteEnabled,
+               name + ' ' + (isRemoteEnabled ? 'should' : 'should not') +
+                 ' be remote enabled');
```

Comment:
What is the indentation rule here? There is a huge one for the first line and a 2 space for the next.